modules/ccs-gcp-customer-procedure.adoc

```diff
@@ -6,21 +6,30 @@
 
 = Required customer procedure
 
-The Customer Cloud Subscription (CCS) model allows Red Hat to deploy and manage {product-title} into a customer's Google Cloud Platform (GCP) project. Red Hat requires several prerequisites to provide these services.
+The Customer Cloud Subscription (CCS) model allows Red{nbsp}Hat to deploy and manage {product-title} into a customer's Google Cloud Platform (GCP) project. Red Hat requires several prerequisites to provide these services.
 [NOTE]
 ====
 The following requirements in this topic apply to {product-title} on {GCP} clusters created using both the service account and Workload Identity Federation authentication type. For additional requirements that apply to the service account authentication type only, see _Service account authentication type procedure_. For additional requirements that apply to the Workload Identity Federation authentication type only, see _Workload Identity Federation authentication type procedure_.
 ====
 
-[WARNING]
-====
-To use {product-title} in your GCP project, the following GCP organizational policy constraints cannot be in place:
+.Prerequisites
+
+Before using {product-title} in your GCP project, confirm that the following organizational policy constraints are configured correctly where applicable:
 
-* `constraints/iam.allowedPolicyMemberDomains` (This policy constraint is supported only if Red Hat's `DIRECTORY_CUSTOMER_IDs C02k0l5e8` and `C04j7mbwl` are included in the allow list. Use this policy constraint with caution).
+* `constraints/iam.allowedPolicyMemberDomains`
+** This policy constraint is supported only if Red{nbsp}Hat's Directory Customer ID's `C02k0l5e8` and `C04j7mbwl` are included in the allowlist.
 * `constraints/compute.restrictLoadBalancerCreationForTypes`
-* `constraints/compute.requireShieldedVm` (This policy constraint is supported only if the cluster is installed with "Enable Secure Boot support for Shielded VMs" selected during the initial cluster creation).
-* `constraints/compute.vmExternalIpAccess` (This policy constraint is supported when installing a private cluster with GCP Private Service Connect (PSC). For all other cases, this policy constraint is supported only after installation).
+** This policy constraint is supported only when creating a private cluster with GCP Private Service Connect (PSC). You must ensure that the `INTERNAL_TCP_UDP` load balancer type is included in the allowlist or excluded from the deny list.
++
+[IMPORTANT]
 ====
+Although the `EXTERNAL_NETWORK_TCP_UDP` load balancer type is not required when creating a private cluster with GCP Private Service Connect (PSC), disallowing it via this constraint will prevent the cluster from being able to create externally accessible load balancers.
+====
+
+* `constraints/compute.requireShieldedVm`
+** This policy constraint is supported only if the cluster is created with "Enable Secure Boot support for Shielded VMs" selected during the initial cluster creation.
```

Comment:
```suggestion
** This policy constraint is supported only if the cluster is created with *Enable Secure Boot support for Shielded VMs* selected during the initial cluster creation.
```
Pretty sure this should be formatted as bold, judging by substep 2 of step 1 in this OCP procedure: https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/installing_on_gcp/installing-gcp-customizations#installation-initializing_installing-gcp-customizations.